reference-implementations/icrc25/README.md

```diff
@@ -0,0 +1,38 @@
+# ICRC-25 Reference Implementation
```

Comment:
Could you make the following changes here?
* Mark the reference implementation as work in progress with a disclaimer that it might not yet implement the latest changes of the standards.
* Add a Todo section with probably the following points:
  * Catch up with reference implementation
  * Extract crypto library for signature verification
  * Improve agent-js to expose the features we need here (to get rid of the copied code)